prototype/p2p/messenger_test.go

```diff
@@ -0,0 +1,78 @@
+package p2p
+
+import (
+	"pocket/p2p/types"
+	"testing"
+)
+
+func TestMessager_Protobuff(t *testing.T) {
```

Comment:
Pro tip: If you want people to run your tests, add a Makefile target to run them.

`go test ./...` will include it, but it's nice to have a dedicated target for it too.